src/test/e2e/PageObjects/HomePage.ts

```diff
@@ -0,0 +1,31 @@
+import { Page } from 'puppeteer';
```

Comment:
can we tag our page object files as `blahblah.po.ts` even though they're in the page object folder youve created, just nice to have them tagged as .po